### PR TITLE
refine node disconnection test

### PR DESCRIPTION
**Pull request by yangchiu**

### CODE REVIEW

File: docs/content/manual/pre-release/node/node-disconnection.md [hunk 1]
```diff
@@ -4,14 +4,58 @@ title: Node disconnection test
...
+4. Keep writing data directly to the volume. (sudo dd if=/dev/urandom of=/dev/longhorn/test-1 bs=2M count=2048)
+5. Disconnect the network of the node that the volume attached to for 100 seconds during the data writing. (sudo nohup ./network_down.sh 100)
+6. Wait for the node back.
+7. The volume should remain detached, and all replicas remain in ERROR state.
 
 ### Case 2:
+1. Disable auto-salvage.
+2. Create a volume named test-1.
+3. Create PV/PVC for the volume through longhorn UI.
+4. Create a pod with the volume. (kubectl apply -f pod.yaml)
```
Comment (innobead) [inline docs/content/manual/pre-release/node/node-disconnection.md:34]: Do we need to create a pod? I thought it should be the same just attaching the volume to a node but w/o writing data during the node not ready.

File: docs/content/manual/pre-release/node/node-disconnection.md [hunk 2]
```diff
@@ -4,14 +4,58 @@ title: Node disconnection test
...
+      args: ["/bin/sh", "-c", "while true;do date;sleep 5; done"]
+      volumeMounts:
+        - name: pod-data
+          mountPath: /data
+  volumes:
+    - name: pod-data
+      persistentVolumeClaim:
+        claimName: test-1
+```
+5. Disconnect the network of the node that the volume attached to for 100 seconds during the data writing (sudo nohup ./network_down.sh 100)
```
Comment (innobead) [inline docs/content/manual/pre-release/node/node-disconnection.md:54]: I thought what you want to describe is, right?

Disconnect the network of the node that the volume attached to for 100 seconds and during the time, there is no any data writing (sudo nohup ./network_down.sh 100)

**Comment by innobead** [comment]
> _ No description provided. _

It's better to have a description for each PR instead of `empty`.

File: docs/content/manual/pre-release/node/node-disconnection.md (hunk 1 above)
Comment (yangchiu) [reply docs/content/manual/pre-release/node/node-disconnection.md]: After consideration and discussion, keep test cases consistent and simple and modify this test case to "just attaching the volume to a node but w/o writing data during the node not ready."

**Review by innobead — APPROVED** [review]
LGTM

**Pull request merged**